.vib/wordpress/cypress/cypress/integration/wordpress_spec.js

```diff
@@ -68,13 +72,16 @@ it('checks if admin can edit a site', () => {
   cy.url().should('include', '/wp-admin/site-editor.php');
 })
 
-it('checks if admin can edit a post', () => {
+it('checks if admin can create a post', () => {
   cy.login();
-  cy.visit('/wp-admin/edit.php');
-  cy.get('#bulk-action-selector-top').should('exist');
-  cy.get('#post-1 .row-title').should('exist').click();
-  cy.url().should('include', '/post.php?post=1&action=edit');
-  cy.get('#editor').should('exist');
+  cy.visit('/wp-admin/post-new.php');
+  cy.get('.components-modal__header > .components-button').click();
+  cy.get('#editor').should('be.visible');
+  cy.contains('button[type="button"]',"Publish").click();
+  cy.get('h1[aria-label="Add title"]').should('be.visible').clear().type(`Test Hello World!${random}`);
+  cy.get('.editor-post-save-draft').should('be.visible').click();
+  cy.get('.editor-post-saved-state').should('exist');
```

Comment:
I think this assertion suits better:

```suggestion
  cy.get('.editor-post-saved-state').should('be.visible');
```
`cy.get()` will already throw an error if the specified element does not exist in the DOM. Additionally, the element may exist but be hidden, so I think this might be more helpful.

Reply:
I made this change because I noticed the `be.visible` assertion failing randomly and adding some flakiness... Let me think about a third way.